src/test/java/com/prgrms/commandLineApplication/voucher/PercentDiscountVoucherTest.java

```diff
@@ -0,0 +1,31 @@
+package com.prgrms.commandLineApplication.voucher;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class PercentDiscountVoucherTest {
+
+  Voucher percentDiscountVoucher;
+
+  @ParameterizedTest
+  @CsvSource(value = {"20|80000", "30|70000", "50|50000"}, delimiter = '|')
+  @DisplayName("percentDiscountVoucher 할인 계산 결과 일치 테스트")
+  void percentDiscountVoucher_계산_성공(double discountValue, double result) {
+    String voucherType = "percent";
+    percentDiscountVoucher = new PercentDiscountVoucher(voucherType, discountValue);
+    assertThat(percentDiscountVoucher.discount(100000)).isEqualTo(result);
+  }
```

Comment:
```suggestion
  void percentDiscountVoucher_계산_성공(double discountValue, double expected) {
    String voucherType = "percent";
    percentDiscountVoucher = new PercentDiscountVoucher(voucherType, discountValue);
    
    double result = percentDiscountVoucher.discount(100000);
    
    assertThat(result).isEqualTo(expected);
  }
```
이렇게 작성하면 더 읽기 편한 것 같아요.

Reply:
😲 훨씬 눈에 잘 들어옵니다!! 이런 부분도 고려해가며 테스트 코드 작성하겠습니다!🤩

Reply:
15990a6431fb73f3a39b18aaee2a4f91aafc2d93